apps/web/pages/team/[slug].tsx

```diff
@@ -249,29 +245,39 @@ function TeamPage({ team, isUnpublished, markdownStrippedBio, isValidOrgDomain }
 export const getServerSideProps = async (context: GetServerSidePropsContext) => {
   const ssr = await ssrInit(context);
   const slug = Array.isArray(context.query?.slug) ? context.query.slug.pop() : context.query.slug;
-  const { isValidOrgDomain } = orgDomainConfig(context.req.headers.host ?? "");
+  const { isValidOrgDomain, currentOrgDomain } = orgDomainConfig(
+    context.req.headers.host ?? "",
+    context.params?.orgSlug
+  );
   const flags = await getFeatureFlagMap(prisma);
-
-  const team = await getTeamWithMembers(undefined, slug);
+  const team = await getTeamWithMembers({ slug, orgSlug: currentOrgDomain });
   const metadata = teamMetadataSchema.parse(team?.metadata ?? {});
-
+  console.warn("gSSP, team/[slug] - ", {
+    isValidOrgDomain,
+    currentOrgDomain,
+    ALLOWED_HOSTNAMES: process.env.ALLOWED_HOSTNAMES,
+    flags: JSON.stringify,
+  });
```

Comment:
Is this intended? @leog

Reply:
Introduced by @alannnc or @hariombalhara, didn’t remember to remove it